std/format.d

```diff
@@ -2658,27 +2658,31 @@ if (is(FloatingPointTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
     {
         static if (is(T == float) || is(T == double) || (is(T == real) && T.mant_dig == double.mant_dig))
         {
-            import std.math : FloatingPointControl;
+            import std.math;
 
             auto mode = RoundingMode.toNearestTiesToEven;
-            switch (FloatingPointControl.rounding)
+
+            // std.math's FloatingPointControl isn't available on all target platforms
+            static if (is(typeof(FloatingPointControl)))
```

Comment:
This needs to be `is(FloatingPointControl)` not `is(typeof(FloatingPointControl))`.

Reply:
Thx, fixed.